src/libpriv/rpmostree-kernel.c

```diff
@@ -341,12 +426,16 @@ rpmostree_run_dracut (int     rootfs_dfd,
                                       &tmpf, error))
     goto out;
 
+  /* If we're rebuilding, we use the *current* /etc so we pick up any modified
+   * config files.  Otherwise, we use the usr/etc defaults.
+   */
   if (rebuild_from_initramfs)
     bwrap = rpmostree_bwrap_new (rootfs_dfd, RPMOSTREE_BWRAP_IMMUTABLE, error,
                                  "--ro-bind", "/etc", "/etc",
                                  NULL);
   else
     bwrap = rpmostree_bwrap_new (rootfs_dfd, RPMOSTREE_BWRAP_IMMUTABLE, error,
+                                 "--ro-bind", "./usr/etc", "/etc",
```

Comment:
Minor/optional: this can just be `usr/etc`, right? (To be more consistent with other relative bindings we do elsewhere).